src/3d/qgs3dmapscene.cpp

```diff
@@ -325,7 +320,6 @@ void Qgs3DMapScene::onCameraChanged()
 
 void Qgs3DMapScene::updateScene()
 {
-  QgsEventTracing::addEvent( QgsEventTracing::Instant, QStringLiteral( "3D" ), QStringLiteral( "Update Scene" ) );
```

Comment:
What's the rationale behind removing this?

Reply:
Ah that was just a random mistake. Sorry about that